RtspServer/ClientSession.cs

```diff
@@ -393,16 +390,15 @@ public void StartReceive()
 
             if (object.ReferenceEquals(m_RtspSocket, null).Equals(false) && HasRuningServer)
             {
-                if (LastRecieve == null)
+                var receiveArgs = new SocketAsyncEventArgs
```

Comment:
Lets not change this yet, SocketAsyncEventArgs is a class and must be disposed only when no longer used, its supposed to support Connect, Send and Receive all from a single instance.